recipes/alphashape/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "alphashape" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4f83279dfccc71c17908161ddfa64f3c56727c10022179f7edec70857c636088
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - alphashape=alphashape.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - click >=6.0
+    - click_log >=0.3.2
+    - pip
+    - python
+    - scipy >=1.0.0
```

Comment:
Beside `pip` and `python`, are you sure you need the other dependencies for installing the package, and not only for running it? I would assume that you only need `pip` and `python` here...

Reply:
I agree, I misunderstood the difference between the two.